tensorflow/python/ops/nn_grad.py

```diff
@@ -338,9 +338,9 @@ def _EluGradGrad(op, grad):
 def _SeluGradGrad(op, grad):
   x = op.inputs[1]
   scale_alpha = 1.7580993408473768599402175208123
-  return (gen_nn_ops._selu_grad(grad, op.outputs[0]),
+  return (gen_nn_ops._elu_grad(grad, op.outputs[0]),
           array_ops.where(
-              x < 0., gen_nn_ops._selu_grad(grad, op.outputs[0] + scale_alpha),
+              x < 0., gen_nn_ops._elu_grad(grad, op.outputs[0] + scale_alpha),
```

Comment:
Changing this like from `_selu_grad` to `_elu_grad` fixes the failing test. Not sure why this works, could someone please explain. Why is `_SeluGradGrad` returning a tuple?